test/pylib/rest_client.py

```diff
@@ -149,6 +150,16 @@ async def get_host_id(self, server_id: str) -> str:
         host_uuid = host_uuid.lstrip('"').rstrip('"')
         return host_uuid
 
+    async def get_host_id_map(self, dst_server_id: str) -> Dict:
+        """Retrieve the mapping of endpoint to host ID"""
+        response = await self.client.get("/storage_service/host_id", dst_server_id)
```

Comment:
Please make a run-time type assertion, similar to the other methods in this class which return stuff (e.g. `get_gossip_generation_number`)

(otherwise mypy will complain)

You should also be able to use lowercase `dict` which doesn't need an `import` (newer Python versions only)

Reply:
fixed